src/Cli/ConfigGenerator.cs

```diff
@@ -716,9 +733,81 @@ private static bool TryConfigureRuntimeOptions(
                 }
             }
 
+            // Cache: Enabled and TTL
+            if (options.RuntimeCacheEnabled != null ||
+                options.RuntimeCacheTTL != null)
+            {
+                EntityCacheOptions? updatedCacheOptions = runtimeConfig?.Runtime?.Cache ?? new();
+                bool status = TryUpdateConfigureCacheValues(options, ref updatedCacheOptions);
+                if (status)
+                {
+                    runtimeConfig = runtimeConfig! with { Runtime = runtimeConfig.Runtime! with { Cache = updatedCacheOptions } };
+                }
+                else
+                {
+                    return false;
+                }
+            }
+
             return runtimeConfig != null;
         }
 
+        /// <summary>
+        /// Attempts to update the Config parameters in the Rest runtime settings based on the provided value.
+        /// Validates that any user-provided parameter value is valid and then returns true if the updated Rest options
+        /// needs to be overwritten on the existing config parameters
+        /// </summary>
+        /// <param name="options">options.</param>
+        /// <param name="updatedRestOptions">updatedRestOptions.</param>
+        /// <returns>True if the value needs to be udpated in the runtime config, else false</returns>
```

Comment:
```suggestion
        /// <returns>True if the value needs to be updated in the runtime config, else false</returns>
```
I also see there are some other comments in this file that have the same error, if you can fix them.

Reply:
done